tools/shared.py

```diff
@@ -318,6 +321,9 @@ def parse_config_file():
     'CLOSURE_COMPILER',
     'JAVA',
     'JS_ENGINES',
+    'WASMER',
+    'WASMTIME',
```

Comment:
I don't think these two are referenced anywhere.

Reply:
I think they could be used in jsrun, so

```
  is_wasmer = 'wasmer' in jsengine
=>
  is_wasmer = engine == shared.WASMER
```

The only reason I didn't do that is the JS engine detection doesn't work that way. But I could refactor them all that way, what do you think?

Also, some tests eventually may only run in wasmer etc., like we have tests that only run in `NODE_JS` etc., so having the variable is useful I think.